app/views/admin/participants/show.html.erb

```diff
@@ -10,37 +10,42 @@
       <tbody class="govuk-table__body">
         <tr class="govuk-table__row">
           <th scope="row" class="govuk-table__header">Full name</th>
-          <td class="govuk-table__cell"><%= @participant.full_name %></td>
+          <td class="govuk-table__cell"><%= @participant_profile.user.full_name %></td>
         </tr>
         <tr class="govuk-table__row">
           <th scope="row" class="govuk-table__header">Email address</th>
-          <td class="govuk-table__cell"><%= @participant.email %></td>
+          <td class="govuk-table__cell"><%= @participant_profile.user.email %></td>
         </tr>
         <tr class="govuk-table__row">
           <th scope="row" class="govuk-table__header">Participant type</th>
-          <td class="govuk-table__cell"><%= @participant.user_description %></td>
+          <td class="govuk-table__cell"><%= t @participant_profile.participant_type, scope: "schools.participants.type" %></td>
         </tr>
       </tbody>
     </table>
   </div>
   <div class="govuk-grid-column-one-half">
     <table class="govuk-table">
       <tbody class="govuk-table__body">
-        <% if @participant.early_career_teacher? %>
+        <% if @participant_profile.ect? %>
           <tr class="govuk-table__row">
             <th scope="row" class="govuk-table__header">Mentor</th>
-            <td class="govuk-table__cell"><%= @participant.early_career_teacher_profile.mentor&.full_name&.presence || 'Not yet assigned' %></td>
+            <td class="govuk-table__cell"><%= @participant_profile.mentor&.full_name || 'Not yet assigned' %></td>
```

Comment:
Empty strings in ruby are truthy, hence I imagine the `presence`.

Although, I guess the `full_name` cannot be an empty string, so I think we are safe. Is that true?

Reply:
I think if the name were blank we wouldn't want to show "not yet assigned", but no, it should never be blank. Will leave this as-is I think